src/main/java/com/softserve/repository/impl/SportHubDatabaseRepositoryImpl.java

```diff
@@ -0,0 +1,53 @@
+package com.softserve.repository.impl;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Repository;
+
+import com.softserve.dto.ArticleDTO;
+import com.softserve.repository.SportHubDatabaseRepository;
+
+/**
+ * 
+ * @author José Castellanos
+ * Implementation of SportHubDatabaseRepository that helps to build complex queries
+ */
+@Repository
+public class SportHubDatabaseRepositoryImpl implements SportHubDatabaseRepository {
+
+	private JdbcTemplate jdbcTemplate;
```

Comment:
I'm not against using jdbcTemplate but usually, we don't use it along JPA. We prefer to use the repositories built-in methods and build the result on top of it.
This is optional

Reply:
I got it, but en this case we made a "complexity" query to map data, because we don't get the database credentials to make DDL udpdats and set the reletionship to using jpa and Joceline wasn't available to help us.